lib/pages/setting_list_view.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:helpwave/services/language_model.dart';
+import 'package:provider/provider.dart';
+
+class SettingListView extends StatelessWidget {
```

Comment:
change Name to "SettingLanguageSelctionPage" and file name to "SettingLanguageSelction"